src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/CodeGeneration/OperatorGenerator.cs

```diff
@@ -48,8 +49,9 @@ internal static OperatorDeclarationSyntax GenerateOperatorDeclaration(
         var declaration = GenerateOperatorDeclarationWorker(method, destination, info, cancellationToken);
         declaration = UseExpressionBodyIfDesired(info, declaration, cancellationToken);
 
-        return AddAnnotationsTo(method,
+        declaration = AddAnnotationsTo(method,
             ConditionallyAddDocumentationCommentTo(declaration, method, info, cancellationToken));
+        return declaration.WithAdditionalAnnotations(Formatter.Annotation);
```

Comment:
similar annotations are applied to method generation.  Previously the override impl was doing additional formatting (beyond the formatter.annotation spans) and so this missing formatting annotation on operators here wasn't noticed.